governance.md

```diff
@@ -0,0 +1,133 @@
+# Principles
+
+The Confidential Containers is an open source organization and community
+that adheres to the following principles:
+
+* Open - Confidential Containers is an open source project licensed under the
+[Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0). We welcome all [contributions](https://github.com/confidential-containers/community/blob/main/CONTRIBUTING.md)
+* Respectful and welcoming - See our [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md)
+* Transparent - All discussions leading to contributions to the project should be open to all and done in public. There maybe exceptions for preliminary effort (e.g. security related work) but the point of disclosure will still lead to a transparent discussion in public, either through our GitHub repositories or public meetings.
+
+# Projects
+
+The Confidential Containers organization is composed of multiple projects.
+
+A project is the primary unit of collaboration, therefore each project has its
+own repository and maintainers team. All projects follow the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+# Community Members and Roles
+
+Everyone is welcome to participate to the Confidential Containers projects,
+and there are 3 community member roles:
+
+## Contributor
+
+Anyone that contributed to one of the Confidential Containers projects within the
+last 12 months is a *Contributor*. Any merged Pull Request is considered a valid
+contribution.
+
+Contributions are not limited to code alone. Adding or enhancing documentation,
+tests, tools, project artifacts are all valuable ways to contribute to a
+Confidential Containers project.
+
+Project contributions will be reviewed by the project maintainers and should
+pass all applicable tests.
+
+## Maintainer
+
+Each project has one or more *Maintainer*. Project maintainers are first and
+foremost active *Contributors* to the project and are responsible for:
+
+* Setting technical directions for the project.
+* Facilitating, reviewing and merging contributions. They have write access to
+  the project repository.
+* Creating and assigning project issues.
+* Enforcing the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+The list of maintainers for a project is defined by the project
+`CODEOWNERS` file.
```

Comment:
:+1: simple enough. Maybe it's worth saying the CODEOWNERS at the top-level of the repo, for the `*` glob?

Because potential drawback is in the future having nested CODEOWNERS to isolate folks for particular subsystem subject-matter-expertise, or different glob patterns than just `*`

Reply:
Done, thanks.